components/gauge-arc/scale.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Scale
+page_title: Scale
+description: Arc Gauge for Blazor - Scale.
+slug: arc-gauge-scale
+tags: telerik,blazor,arc,gauge,scale
+published: True
+position: 5
+---
+
+## Arc Gauge Scale
+
+The scale of the arc gauge renders the values, pointers and labels. You can customize it by adding an instance of the `<ArcGaugeScale>` to the `<ArcGaugeScales>` collection, child tag of the `<TelerikArcGauge>`. The `<ArcGaugeScale>` exposes the following parameters:
+
+* [Min and Max](#min-and-max)
+
+* [MinorUnit and MajorUnit](#minorunit-and-majorunit)
+
+* [StartAngle and EndAngle](#startangle-and-endangle)
+
+* [Reverse](#reverse)
+
+* [Additional Customization](#additional-customization)
+
+    * [Example: Enable the MinorUnit ticks](#example-enable-the-minorunit-ticks)
+
+
+## Min and Max
+
+* The `Max` (`double`) parameter controls the maximum value that the component can reach.
+
+* The `Min` (`double`) parameter controls the lowest value of the component.
+
+>caption Change the lowest and the highest values for the scale. The result from the code snippet below.
+
+![Min and max parameters example](images/min-and-max-arc-gauge.png)
+
+````CSHTML
+@* Use the Min and Max parameters to change the lowest and highest values for the scale *@
+
+<TelerikArcGauge>
+    <ArcGaugeScales>
+        <ArcGaugeScale Min="10" Max="100">
+            <ArcGaugeScaleLabels Visible="true" />
+        </ArcGaugeScale>
+    </ArcGaugeScales>
+
+    <ArcGaugePointers>
+
+        <ArcGaugePointer Value="30">
+        </ArcGaugePointer>
+
+    </ArcGaugePointers>
+</TelerikArcGauge>
+````
+
+## MinorUnit and MajorUnit
+
+* The `MajorUnit` (`double`) parameter controls the interval between the major unit divisions of the component. The values provided to the `ArcGaugePointer` will render as a `MajorUnit` tick. The [labels]({%slug arc-gauge-labels%}) will be rendered next to the `MajorUnit` ticks.
+
+* The `MinorUnit` (`double`) parameter controls the interval between the minor unit divisions of the component.
+
+>caption Change the rendering frequency of the major unit divisions. The result from the code snippet below.
+
+![Minor and major units parameters](images/minor-and-major-units-arc-gauge.png)
+
+````CSHTML
+@* Update the rendering of the major ticks *@
+
+<TelerikArcGauge>
+    <ArcGaugeScales>
+        <ArcGaugeScale Min="0" Max="100" MajorUnit="10">
+            <ArcGaugeScaleLabels Visible="true" />
+        </ArcGaugeScale>
+    </ArcGaugeScales>
+
+    <ArcGaugePointers>
+
+        <ArcGaugePointer Value="30">
+        </ArcGaugePointer>
+
+    </ArcGaugePointers>
+</TelerikArcGauge>
+````
+
+## StartAngle and EndAngle
```

Comment:
People will want to make a full circle gauge out of this. If it is possible (e.g., start angle = 0 and end angle = 360) add such an example. Otherwise, add a link to the circular gauge (could be a green tip above or below the example)